### ISSUE TITLE
Favicon has low color contrast in dark mode (community feedback)

### ISSUE THREAD

**Issue by alexisreytech** [accessiblecommunity/accessiblecommunity.github.io#293]
In the browser tab in dark mode, the favicon (Accessible Community logo) does not have sufficient color contrast.

Here is an image of the favicon in dark mode:
<img width="162" alt="ac-favicon" src="https://github.com/accessiblecommunity/accessiblecommunity.github.io/assets/89283739/00c05d54-34e7-43c4-85cf-0d33f9f209e9">
